modules/run-nomad/run-nomad

```diff
@@ -322,16 +322,6 @@ function run {
         user="$2"
         shift
         ;;
-      --cluster-tag-key)
-        assert_not_empty "$key" "$2"
-        cluster_tag_key="$2"
-        shift
-        ;;
-      --cluster-tag-value)
-        assert_not_empty "$key" "$2"
-        cluster_tag_value="$2"
-        shift
-        ;;
```

Comment:
Again, why remove?

Reply:
In this `run-nomad` script, these variables are defined but never referenced. I believe they were [copy pasted from run-consul](https://github.com/hashicorp/terraform-aws-consul/blob/master/modules/run-consul/run-consul#L490) where they are [used to configure Consul to discover its peers via tags](https://github.com/hashicorp/terraform-aws-consul/blob/master/modules/run-consul/run-consul#L642)

Reply:
Agree, when debugging, these variables initially confused me on how they were forming the cluster. It was only after reading the consul strategy did I realise the tags for nomad cluster forming are actually not important in this case.